packages/client/package.json

```diff
@@ -52,5 +53,6 @@
   ],
   "main": "./build/js/index.js",
   "module": "./build/js/index.mjs",
-  "types": "./build/types/index.d.ts"
+  "types": "./build/types/index.d.ts",
+  "type": "module"
```

Comment:
We want to not do this, right?

Reply:
I'm okay with it in `client` and it seems to work fine in my legacy check.

Reply:
This is going to make it hard for anyone downstream to consume the package, just like we hit with conjure-lite